import/index.html

```diff
@@ -292,6 +309,91 @@
         return await crypto.subtle.verify({ name: "ECDSA", namedCurve: "P-256", hash: {name: "SHA-256" }}, quorumKey, publicSignatureBuf, publicKeyBuf);
       }
 
+      /**
+       * Function to validate and sanitize the styles object using the accepted map of style keys and values (as regular expressions).
+       * Any invalid style throws an error. Returns an object of valid styles.
+       * @param {Object} styles
+       * @return {Object}
+       */
+       function validateStyles(styles, element) {
+        const validStyles = {};
+
+        const cssValidationRegex = {
+          "padding": "^(\\d+(px|em|%|rem) ?){1,4}$",
+          "margin": "^(\\d+(px|em|%|rem) ?){1,4}$",
+          "borderWidth": "^(\\d+(px|em|rem) ?){1,4}$",
+          "borderStyle": "^(none|solid|dashed|dotted|double|groove|ridge|inset|outset)$",
+          "borderColor": "^(transparent|inherit|initial|#[0-9a-f]{3,8}|rgba?\\(\\d{1,3}, \\d{1,3}, \\d{1,3}(, \\d?(\\.\\d{1,2})?)?\\)|hsla?\\(\\d{1,3}, \\d{1,3}%, \\d{1,3}%(, \\d?(\\.\\d{1,2})?)?\\))$",
+          "borderRadius": "^(\\d+(px|em|%|rem) ?){1,4}$",
+          "fontSize": "^(\\d+(px|em|rem|%|vh|vw|in|cm|mm|pt|pc|ex|ch|vmin|vmax))$",
+          "fontWeight": "^(normal|bold|bolder|lighter|\\d{3})$",
+          "fontFamily": "^[^\";<>]*$", // checks for the absence of some characters that could lead to CSS/HTML injection
```

Comment:
Nit: it'd be nice to have some unit tests asserting that these characters aren't allowed? You already have a harness for tests, let's use it!